discord_bots/commands.py

```diff
@@ -2606,15 +2579,9 @@ async def showgame(ctx: Context, game_id: str):
 
 
 @bot.command()
+@commands.check(is_admin)
 async def showgamedebug(ctx: Context, game_id: str):
```

Comment:
i don't really want this command to be available to admins. it shows other players stats. would prefer to comment this out or only enable in debug mode or something

Reply:
Gotcha, yeah that makes sense. I can add the is_owner decorator in the future